solutions/02-follower-reads/movr/pricing/data/pricing_database.sql

```diff
@@ -1,8 +1,8 @@
 CREATE DATABASE movr_pricing;
 
-ALTER DATABASE movr_pricing SET PRIMARY REGION "us-west";
+ALTER DATABASE movr_pricing SET PRIMARY REGION "us-east";
 ALTER DATABASE movr_pricing ADD REGION "us-central";
-ALTER DATABASE movr_pricing ADD REGION "us-east";
+ALTER DATABASE movr_pricing ADD REGION "us-west";
 
```

Comment:
I know this is default but to keep consistent we need to add:

ALTER DATABASE movr_pricing SURVIVE REGION FAILURE;

Reply:
The solution 4 file also needs to have this added.

Reply:
added to those solutions!